src/App.js

```diff
@@ -1,14 +1,21 @@
-import logo from './logo.svg';
 import './App.css';
-import SearchForm from './components/searchForn/searchForm';
+import SearchForm from './components/searchForm/searchForm';
 import GenreSelect from './components/genreSelect/genreSelect';
 import Counter from './components/counter/counter';
 function App() {
+  const genres = ['Action', 'Comedy', 'Drama', 'Science Fiction', 'Horror'];
+  const selectedGenre = 'Drama';
+  const handleSearch = (searchTerm) => {
+    console.log("Searched with: " + searchTerm);
+  }
+  const handleGenreSelect = (selectedGenre) => {
+    console.log("Selected Genre: " + selectedGenre);
+  }
   return (
     <div>
-    <SearchForm/>
-    <GenreSelect/>
-    <Counter/>
+      <SearchForm initialSearchTerm='Movie' onChange={handleSearch} />
+      <GenreSelect genres={genres} selectedGenre={selectedGenre} onChange={handleGenreSelect} />
+      <Counter initialCount={3} />
```

Comment:
No magic number please, use a constant file or declare a variable and initialize.

Reply:
Makes sense. Moved to a constant file.